__WIP_pg_schema_review/prisma/dbml/schema.dbml

```diff
@@ -0,0 +1,147 @@
+//// ------------------------------------------------------
+//// THIS FILE WAS AUTOMATICALLY GENERATED (DO NOT MODIFY)
+//// ------------------------------------------------------
+
+Table block {
+  hash String [pk, unique]
+  number Int [unique, not null]
+  tokens token [not null]
+}
+
+Table token {
+  id String [pk, unique]
+  token_id String [not null]
+  mint_time String [not null]
+  token_asset_id String [not null]
+  token_asset token_asset [not null]
+  blocks block [not null]
+  token_contract_id String [not null]
+  contract token_contract [not null]
+  owner_id String [not null]
+  imported erc721_import_result [not null]
+}
+
+Table token_asset {
+  id String [pk, unique]
+  referrers token [not null]
+  token_uri String [unique, not null]
+  ipnft String [not null]
+  problem String
+  metadata metadata
+}
+
+Table metadata {
+  id String [pk, unique]
+  cid String
+  source_id String [unique, not null]
+  source token_asset [not null]
+  name String
+  description String
+  imageId String [not null]
+  image_uri String [not null]
+  assets resource [not null]
+}
+
+Table resource {
+  referrers metadata [not null]
+  status resource_status [not null]
+  status_text String [not null]
+  created DateTime [not null]
+  updated DateTime [not null]
+  uri String [pk, unique]
+  ipfs_url String [unique, not null]
+  problem String [not null]
+  cid String
+  content content
+}
+
+Table content {
+  cid String [pk, unique]
+  resources resource [not null]
```

Comment:
Is there way to visualize fields that aren't actually in the table somehow differently ? Looking at .sql I see there is no `resources` table, but looking at the diagram they look like actual fields https://dbdiagram.io/d/613293c9825b5b0146f27f32

Reply:
I'll look into this

Reply:
this might be what we need in this case:  https://github.com/ipfs-shipyard/nft.storage/pull/394#issuecomment-913334271

You can see whats in the actual table vs the 'prisma api'.  for the short term we just care about the actual tables, I've converted all the joins into explicit join tables, so I think it will be a lot easier to read now.